skfda/basis.py

```diff
@@ -2208,7 +2208,10 @@ def __add__(self, other):
                                   self.coefficients + other.coefficients)
 
         coefs = self.coefficients.copy()
-        coefs[:, 0] = self.coefficients[:, 0] + numpy.array(other)
+        try:
+            coefs[:, 0] = self.coefficients[:, 0] + numpy.array(other)
+        except:
```

Comment:
Catch ONLY the exception that is thrown in the conversion.

Reply:
We talk that the conversion error will be thrown not caught.

The add mismatch will be treated as an exception